zenodo/modules/records/api.py

```diff
@@ -52,9 +58,57 @@ def dumps(self):
         return self.data
 
 
-class ZenodoFilesIterator(FilesIterator):
-    """Zenodo files iterator."""
+class MetaFilesMixin(object):
```

Comment:
This can be renamed to `ZenodoFilesMixin`, since it overrides `files` and adds `metafiles`. Then below `ZenodoRecord`, should inherit from `invenio_records.api.Record` and `ZenodoFilesMixin`